### ISSUE TITLE
Allow Tz.WatchPeriodically to accept and call a function when triggered.

### ISSUE THREAD

**Issue by OldhamMade** [mathieuprog/tz#10]
Since using `Tz.UpdatePeriodically` on memory-restricted platforms may not be possible, it would be useful if `Tz.WatchPeriodically` could call a function whenever it logs that there is an update. This would allow users of the library to do something proactive, like trigger an email or SMS.

**Comment by mathieuprog** [comment]
I added the following options to `Tz.WatchPeriodically` (not yet published)

```elixir
{
  Tz.WatchPeriodically, 
    interval_in_days: 5,
    on_update: fn version ->
      IO.inspect "New update available #{version}!"
    end
}
```

Is the naming `on_update` okay?

**Comment by mathieuprog** [comment]
Released in v0.21. Let me know if you have any feedback.
